packages/ui/src/components/va-color-input/VaColorInput.vue

```diff
@@ -1,11 +1,10 @@
 <template>
   <div class="va-color-input">
     <va-input
-      class="va-color-input__input"
-      placeholder="input color"
+      v-bind="vaInputProps"
       v-model="valueComputed"
+      class="va-color-input__input"
       :tabindex="tabIndexComputed"
-      :disabled="$props.disabled"
     >
```

Comment:
max-width for default color input 120px
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/ae261d6e-72ba-412b-875f-c1bd8581eaa9)